pkg/config/config.go

```diff
@@ -112,16 +114,28 @@ type SecurityCapabilityLists struct {
 
 // ParseFile parses config from a file.
 func ParseFile(path string) (Configuration, error) {
-	configBox := packr.New("Config", "../../examples")
-	var rawBytes []byte
 	var err error
+	var rawBytes []byte
+	configBox := packr.New("Config", "../../examples")
 	if path == "" {
```

Comment:
This won't work as-is. If path is `""`, it will still hit the `else` block below, and overwrite `rawBytes`. You'll see it fail if you run `go run main.go --dashboard --dashboard-port 3000 --audit-path ./examples`

If you change the `if strings.HasPrefix` line to an `else if`, that should fix the issue.